portfolio/src/main/java/com/google/sps/servlets/Review.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+public class Review {
```

Comment:
Nice! Can we add some class-level documentation/comments above this describing what the 'Review' class is/what it does?